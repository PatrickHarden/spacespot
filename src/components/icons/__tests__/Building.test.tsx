import React from 'react'
import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'

import Comp from '../Building'

describe('Building icon', () => {
  it('renders without crashing', async () => {
    const wrapper = mount(<Comp />)
    expect(mountToJson(wrapper)).toMatchSnapshot()
  })
})
