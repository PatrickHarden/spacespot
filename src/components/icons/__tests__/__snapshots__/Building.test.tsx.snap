// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Building icon renders without crashing 1`] = `
<Building>
  <svg
    height={58}
    width={50}
  >
    <title>
      flat
    </title>
    <g
      fill="none"
      fillRule="nonzero"
    >
      <path
        d="M42.457 55H26V11h18v36.767l-1.543 2.417z"
        fill="#FF4E01"
        fillOpacity={0.7}
      />
      <path
        d="M28 5h-2.738L23 24.26 25.262 55 28 54.59z"
        fill="#FDAE02"
      />
      <path
        d="M2.773 0v4.885H0V55.13L15.676 57 26 55.545V0z"
        fill="#FDCB02"
      />
      <g
        fill="#F3FFFB"
      >
        <path
          d="M8.078 11.06h3.397v4.838H8.078zM16.534 11.06h3.397v4.838h-3.397zM8.078 20.735h3.397v4.837H8.078zM16.534 20.735h3.397v4.837h-3.397zM8.078 30.41h3.397v4.836H8.078zM16.534 30.41h3.397v4.836h-3.397zM8.078 40.083h3.397v4.837H8.078zM16.534 40.083h3.397v4.837h-3.397z"
        />
      </g>
      <path
        d="M33 21h3v5h-3zM33 31h3v5h-3z"
        fill="#FFD27B"
      />
      <path
        d="M41 48h3v8h-3z"
        fill="#8F4D24"
      />
      <path
        d="M42.5 52c-4.135 0-7.5-3.365-7.5-7.5 0-4.136 3.365-7.5 7.5-7.5 4.136 0 7.5 3.364 7.5 7.5 0 4.135-3.364 7.5-7.5 7.5z"
        fill="#A8D32A"
      />
      <path
        d="M50 55H26.19L25 56.95 26.19 58H50z"
        fill="#8D9CA8"
      />
      <path
        d="M0 55h26v3H0z"
        fill="#AAB8C7"
      />
    </g>
  </svg>
</Building>
`;
